system/jlib/jlog.hpp

```diff
@@ -59,16 +59,17 @@ typedef enum
  * LOG MESSAGE CLASS:                                                                   */
 typedef enum
 {
-    MSGCLS_unknown     = 0x00, // Invalid/unknown log message class
-    MSGCLS_disaster    = 0x01, // Any unrecoverable or critical system errors
-    MSGCLS_error       = 0x02, // Recoverable/not critical Errors
-    MSGCLS_warning     = 0x04, // Warnings
-    MSGCLS_information = 0x08, // Config, environmental and internal status  info
-    MSGCLS_progress    = 0x10, // Progress of workunits. Status of file operations
-    MSGCLS_metric      = 0x20, // A metric line
-    MSGCLS_addid       = 0x40, // Internal use within log system
-    MSGCLS_removeid    = 0x80, // Internal use within log system
-    MSGCLS_all         = 0xFF  // Use as a filter to select all messages
+    MSGCLS_unknown     = 0x000, // Invalid/unknown log message class
+    MSGCLS_disaster    = 0x001, // Any unrecoverable or critical system errors
+    MSGCLS_error       = 0x002, // Recoverable/not critical Errors
+    MSGCLS_warning     = 0x004, // Warnings
+    MSGCLS_information = 0x008, // Config, environmental and internal status  info
+    MSGCLS_progress    = 0x010, // Progress of workunits. Status of file operations
+    MSGCLS_metric      = 0x020, // A metric line
+    MSGCLS_addid       = 0x040, // Internal use within log system
+    MSGCLS_removeid    = 0x080, // Internal use within log system
+    MSGCLS_all         = 0x0FF, // Use as a filter to select all messages except those below
+    MSGCLS_elevated    = 0x800, // Use for component elevated logging
```

Comment:
it still feels wrong to define a bitmap value past the "ALL" mask, for record keeping sake, why aren't we expanding MSGCLS_all to 0xFFF.
also why isn't MSGCLS_elevated = 0x100,